dev/src/components/NavFavorites.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import { useAppState,  useActions } from "../overmind";
+import { Link } from 'react-router-dom'
+import NavBarCourse from "./navbar/NavBarCourse";
+import { isEnrolled,  isVisible } from "../Helpers";
+
+const NavFavorites = ():JSX.Element  =>{
+    const state = useAppState()
+    const actions = useActions()
+    
+    const visible = state.enrollments.filter(enrollment => isEnrolled(enrollment) && isVisible(enrollment))
+    
+    const courses = visible.map((enrollment) => {
+        return <NavBarCourse key={enrollment.getId()} enrollment={enrollment} />
+    })
+
+    const onCourseClick  = () => {
+        actions.setActiveFavorite(!state.showFavorites)
+    }
+    
+    return (
+        <nav className="navigator">
+            <ul key="list" className="SidebarList">
```

Comment:
Are class names supposed to be capitalized ?

Reply:
We changed it to lowercase for the sake of consistency